spring-session/src/main/java/org/springframework/session/MapSession.java

```diff
@@ -40,7 +40,7 @@
  * @since 1.0
  * @author Rob Winch
  */
-public final class MapSession implements ExpiringSession, Serializable {
+public class MapSession implements ExpiringSession, Serializable {
```

Comment:
We use is in a version with FST serialization :) I can make it final but it means each custom implementation will probably copy 90% of this code, as it covers most usecases.